src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.Scanner;
+
 public class Duke {
+    
+    public static final String DIVIDER = "____________________________________________________________\n";
+    
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        
+        Tasks list = new Tasks();
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        
+        while (!line.equalsIgnoreCase("bye")) {
+            System.out.print(DIVIDER);
+            runCommand(list, line);
+            System.out.println(DIVIDER);
+            line = in.nextLine();
+        }
+        exit();
+    }
+    public static void greet() {
+        String greet = DIVIDER +
+                "Hello! I'm Bob\n" +
+                "What can I do for you?\n" +
+                DIVIDER;
+        System.out.println(greet);
+    }
+    
+    private static void runCommand(Tasks list, String line) {
+        String[] commandAndArg = splitCommandAndArgs(line);
+        String command = commandAndArg[0];
+        String arg = commandAndArg[1];
+        
+        switch(command) {
+        case "list":
+            list.printList();
+            break;
+        case "help":
+            printHelp();
+            break;
+        case "mark":
+            runMark(list, arg);
+            break;
+        case "unmark":
+            runUnmark(list, arg);
+            break;
+        case "todo":
+            addTodo(list, arg);
+            break;
+        case "event":
+            addEvent(list, arg);
+            break;
+        case "deadline":
+            addDeadline(list, arg);
+            break;
+        default:
+            printUnknownCommandMessage();
+            printHelp();
+            break;
+            
+        }
+    }
+    
+    /**
+     * Splits line from user input into an array of 2 Strings: Command and Argument separately
+     * If no argument, second element to be ""
+     *
+     * @param line Input from user
+     * @return String array of size 2 containing Command and Argument
+     */
+    private static String[] splitCommandAndArgs(String line) {
+        final String[] splitStrings = line.split(" ", 2);
+        return splitStrings.length == 2 ? splitStrings : new String[]{splitStrings[0], ""};
+    }
+    
+    private static void printHelp() {
+        // TODO
+        System.out.println("list, todo, event, deadline, mark, unmark");
```

Comment:
Perhaps adding more details to the help message on how to use the different commands would be useful.